pallets/subtensor/src/lib.rs

```diff
@@ -363,6 +363,9 @@ pub mod pallet {
     #[pallet::storage] // --- MAP ( hot ) --> cold | Returns the controlling coldkey for a hotkey.
     pub type Owner<T: Config> =
         StorageMap<_, Blake2_128Concat, T::AccountId, T::AccountId, ValueQuery, DefaultAccount<T>>;
+    #[pallet::storage] // --- MAP ( cold ) --> Vec<hot> | Returns the vector of hotkeys controlled by this coldkey.
+    pub type Owned<T: Config> =
```

Comment:
I would probably call this `OwnedHotkeys` to make the intent clear.

Reply:
Done